src/Xml/XmlProductFeedWriter.php

```diff
@@ -36,16 +36,6 @@ public function close(ProductFeedMetadata $metadata)
         $writer = $this->writer;
         $writer->endElement(); // products
 
-        $writer->startElement('metadata');
-        $writer->writeElement('platform', $metadata->getPlatform());
-        $writer->writeElement('agent', $metadata->getAgent());
```

Comment:
[Error] I think we wanted to eliminate only the date and NOT the entire metadata block.

The platform & agent are used to gain more insight into the feed, without it we would know we were dealing with an `Xmlv1` feed (which is our proprietary feed type, that this library helps generate), but we would not know that we were dealing with a Magento v2, for example.

I'll provide a fix. After some discussions, we came to the agreement to remove everything, except the first 2 (platform & agent), since the rest is not used.

Reply:
Also, there is no need to fork the project. If you look at the past PRs from Ronan or me, we push directly to the central repo. We do this for all smaller libraries and we keep the forking workflow dedicated to the "bigger" projects. Thanks.